app/person-escort-record/app/new/controllers.js

```diff
@@ -22,7 +22,7 @@ class NewPersonEscortRecordController extends FormWizardController {
 
   async saveValues(req, res, next) {
     try {
-      req.record = await personEscortRecordService.create(req.move.profile.id)
+      req.record = await personEscortRecordService.create(req.move.id)
```

Comment:
This is the only current usage of `move.profile.id`. There's already a lot of usage of `move.id` 

I'm wondering, there's other usage of `move.profile` throughout the system - do any of these need to be updated to use data off the `move` as opposed to the `profile` ?

Reply:
I'm pretty sure any existing implementations using move ID and profile ID should be okay left as is.

This change is quite specifically related to the way Person Escort Records now need to be created via the API.

Previously, the API expected the profile ID, but, with the new requirement to import information from NOMIS the API needs to know the origin type (if a Prison, import from NOMIS). However, we have a current business rule that will allow profiles to be used across multiple moves, so the API would not be able to determine that if the client application only send the profile ID.

This change essentially unlocks the API to be able to determine its logical correctly. This will be the final piece that will allow the API to start pulling information from NOMIS and I think I'm going to extract this change so that the other functionality can be merged separately.

Reply:
> extract this change so that the other functionality can be merged separately.

Good call - I was thinking of suggesting the same - that way if there are any side effects, it'll be easier to see what caused it.